client/my-sites/checkout/checkout-thank-you/domains/thank-you-content/domain-registration.tsx

```diff
@@ -41,7 +41,7 @@ const DomainRegistrationThankYouProps = ( {
 						),
 						stepCta: (
 							<FullWidthButton
-								href={ domainMappingSetup( selectedSiteSlug, domain ) }
+								href={ domainManagementEdit( selectedSiteSlug, domain, null ) }
```

Comment:
I think what we want is to take the user to `domainManagementList()` instead. This also fits in with the existing text.

Let's use the link to the site domains list page and leave the text as-is for the moment.